rdr_service/services/consent/files.py

```diff
@@ -1058,9 +1058,11 @@ def get_signature_on_file(self):
 
         signature_label_strings = [
             "Participant's Name (printed)",
+            "Participant’s Name (printed)",
```

Comment:
okay to have `Particpant's Name (printed)` twice in this list?

Reply:
its the apostrophe, lol it had me for a second too

Reply:
haha...i see it now!

Reply:
🫠